index.md

```diff
@@ -86,7 +86,7 @@ def connect() {
 }
 {% endhighlight %}
 
-A `MongoDriver` instance manages an actor system; a `connection` manages a pool of connections. In general, MongoDriver or create a MongoConnection are never instantiated more than once. You can provide a list of one ore more servers; the driver will guess if it's a standalone server or a replica set configuration. Even with one replica node, the driver will probe for other nodes and add them automatically.
```

Comment:
This fixes is removing "create". The sentence with "create" is deleted line and new line displayed in green color in this link: https://github.com/ReactiveMongo/reactivemongo-site/pull/11/files#r32109344